src/main/java/jenkins/branch/BranchEventCause.java

```diff
@@ -49,10 +49,13 @@
 
     private final String description;
 
+    private final Object event;
```

Comment:
would be better as `SCMEvent`... but irrespective, there is an issue with serialization as `SCMEvent` contract does not include serializability, so we may not be able to serialize it to store it